src/index.js

```diff
@@ -35,6 +35,7 @@ export default function supernova(env) {
       }, []);
 
       const layout = useStaleLayout();
+      console.log(layout);
```

Comment:
remember to remove all logs before merging

Reply:
done